src/connectors/aws_config.py

```diff
@@ -0,0 +1,348 @@
+"""AWS Config
+Collects AWS EC2 config information into a columnar table
```

Comment:
This collects more than just EC2 data; it also collects security groups, elbs, and the raw snapshots.

Reply:
I am struggling on using this vs actual Config data snapshots...should this be a different connector?

Reply:
It might be better to break these out into three individual connectors, and then also have a Config connector that ingests snapshots and creates tables for useful things that are in snapshots - that way, people can ingest precisely what they want, or they can get it all with configs.

Reply:
Will do!

Reply:
Sorry to change this up; andrey suggested that keeping it as one connector, but allowing you to select (through a drop down) which logs you want to ingest is probably the ideal solution, similar to what's been implemented in the Azure connector. That way, we don't end up with a dozen AWS connectors in the dashboard, with one for each service. Does that seem reasonable to you?

Reply:
Totally, I agree with it as well. otherwise you have too much to configure at once. Will have a first pass by tomorrow.

Reply:
Ok after further investigation, I think it makes sense to separate this into 2 connectors, the Boto3-based asset inventory for EC2, Security Groups, and ELBS, and then another for aws config ingest. The main reason is that the configuration for the connectors are different. AWS config will setup a stage + pipe + landing table + stream + task, therefore requiring a similar configuration to cloudtrail. The Boto3-based inventory will need only an id and secret key. 

Also I would like to also make the assumption that Config will log to a single bucket for all regions in an account, otherwise there would be the need to create a lot of connectors to ingest the data. @greg-snowflake @andrey-snowflake let me know what you think.